src/core/index.js

```diff
@@ -113,8 +114,9 @@ export default class Lieu {
     /**
      * Return value from currentLanguage.locales or null
      * @param localeKey<String>
+     * @param options<Object>
      */
-    localize(localeKey) {
+    localize(localeKey, options) {
```

Comment:
Let's rename the `localize()` function to `trans()`. Should be a separate commit.